### PR TITLE
PCHR-1356: Replace Government ID Field for NI / SSN field

### PR DESCRIPTION
**Pull request by MiyaNoctem**
## Problem
There was a pseudo-field being shown on contacts called "Government ID" which was being used to create a record for the contact in the Identity custom group.  The requirement now was to replace the pseudo-field for a proper custom field associated directly to Individuals.

before

### Backend 
Implemented upgrade that ceates custom group "Inline Custom Data" and "NI / SSN" field. Also, added JS functionality that moves fields on this group to Contact Details group. Removed backend logic that added Government ID to contact edit view.

### Frontend
Remove, `colspan="3"` form  customFields `td` in "hrui.js" and override styles for `custonFields' width`. Added BackstopJS scenarios in `org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-individual.scss`

after

1. Fixed size of "NI / SSN" field:
screen shot 2017-03-06 at 6 30 31 pm

2. BackstopJS Scenarios Test:
screencapture-file-users-bmn-buildkit-build-hr16-sites-all-modules-civicrm-tools-extensions-civihr-uk-co-compucorp-civicrm-hrcore-backstop_data-html_report-index-html-1488803890531

### CODE REVIEW

**Comment by coldrunKacper** [comment]
Approved.

File: uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/edit-form.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var Promise = require('es6-promise').Promise;
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/edit-form.js]: Please call this file `contact-edit-form.js`

Comment (bmnepali) [inline uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/edit-form.js]: will do.

**Comment by coldrunKacper** [comment]
Approved.

**Comment by MiyaNoctem** [comment]
Had to rebase because of conflict in js/dist/hrui.min.js file.

**Pull request merged**